src/targets/public/main.jsx

```diff
@@ -0,0 +1,72 @@
+/* global cozy */
+import React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import { Router, Route, Redirect, hashHistory } from 'react-router'
+import { I18n } from 'cozy-ui/react/I18n'
+
+import configureStore from '../..//store/configureStore'
+import PublicLayout from '../../components/PublicLayout'
+
+import LightFolderView from '../../components/LightFolderView'
+
+document.addEventListener('DOMContentLoaded', init)
+
+const arrToObj = (obj = {}, [key, val = true]) => {
+  obj[key] = val
+  return obj
+}
+
+const getQueryParameter = () => window
+  .location
+  .search
+  .substring(1)
+  .split('&')
+  .map(varval => varval.split('='))
+  .reduce(arrToObj, {})
```

Comment:
I believe you could use [`URL.searchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams) here?